server/server/session/src/main/java/com/alipay/sofa/registry/server/session/push/PushProcessor.java

```diff
@@ -100,7 +110,12 @@ protected List<PushTask> createPushTask(
       MultiSubDatum datum) {
     PushTask pushTask = new PushTaskImpl(pushCause, addr, subscriberMap, datum);
     // set expireTimestamp, wait to merge to debouncing
-    pushTask.expireAfter(sessionServerConfig.getPushDataTaskDebouncingMillis());
+    if (null != pushEfficiencyImproveConfig
```

Comment:
i think we should call fetchPushEfficiencyConfigService.fetchxxx() to get a mill directly, instead of use inAppSBF in PushProcessor.

Reply:
adjusted